deploy/helm/azure-industrial-iot/values.yaml

```diff
@@ -74,7 +74,6 @@ azure:
   # The following CosmosDB containers will be created (<db_name>.<container_name>):
   #   - iiot_opc.iiot_opc-indices
   #   - iiot_opc.iiot_opc-requests
-  #   - OpcVault.AppsAndCertRequests
```

Comment:
@marcschier, this value is still used in `components/opc-ua/src/Microsoft.Azure.IIoT.OpcUa.Registry/src/Migration/VaultApplicationMigration.cs`, which is not removed by this PR. The migration job will still create this CosmosDB container. So this value is still relevant.